war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -996,7 +996,13 @@ function renderOnDemand(e, callback, noBehaviour) {
   if (!e || !e.classList.contains("render-on-demand")) {
     return;
   }
-  var proxy = eval(e.getAttribute("proxy"));
+
+  let proxyMethod = e.getAttribute("data-proxy-method");
+  let proxyUrl = e.getAttribute("data-proxy-url");
+  let proxyCrumb = e.getAttribute("data-proxy-crumb");
+  let proxyUrlNames = e.getAttribute("data-proxy-url-names").split(",");
```

Comment:
shouldn't these be `const`?
```suggestion
  const proxyMethod = e.getAttribute("data-proxy-method");
  const proxyUrl = e.getAttribute("data-proxy-url");
  const proxyCrumb = e.getAttribute("data-proxy-crumb");
  const proxyUrlNames = e.getAttribute("data-proxy-url-names").split(",");
```